Makefile

```
@@ -1,30 +1,30 @@
 
-COMPOSE_DEV = docker compose -f docker/docker-compose.yml
-COMPOSE_PROD = docker compose -f docker/docker-compose.prod.yml
+COMPOSE_DEV = docker compose -f docker-compose.yml
+COMPOSE_PROD = docker compose -f docker-compose.prod.yml
 
 .PHONY: setup build dev prod lint
 
-setup: ## docker/.env作成（初回のみ。パスワード/シークレット類はランダム生成する）
-	@if [ -f docker/.env ]; then \
-		echo "docker/.env は既に存在するため作成をスキップします"; \
+setup: ## .env作成（初回のみ。パスワード/シークレット類はランダム生成する）
```

checkmake (0.3.2)

[warning] Target body for "setup" exceeds allowed length of 5 lines (11).

(maxbodylength)